### PR TITLE
Added better quick start guide, explaining difference between the sub…

### PR DESCRIPTION
**Pull request by sksamuel**
…projects and modules

### CODE REVIEW

**Review by ashishkujoy — COMMENTED** [review]
Looks good to me. Just a thought should we add a separate page dedicated to all type of Kotest modules such as kotest-assertion-json, kotest-assertion-klock, kotest-assertion-sql, kotest-assertion-ktor etc along with the junit runner, and assertion module, so that people will know that they need to add those dependencies in their build file.

File: doc/reference.md [hunk 1]
```diff
@@ -97,7 +117,7 @@ For maven you must configure the surefire plugin for junit tests.
 </plugin>
 ```
 
-And then add the Kotest JUnit5 runner to your build.
+And then add the Kotest JUnit5 runner to your build to use the framework product.
```
Comment (LeoColman) [inline doc/reference.md:120]: I think we want to recommend the console runner even more than JUnit5, perhaps we should suggest it?

**Review by LeoColman — APPROVED** [review]

**Comment by sksamuel** [comment]
> Looks good to me. Just a thought should we add a separate page dedicated to all type of Kotest modules such as kotest-assertion-json, kotest-assertion-klock, kotest-assertion-sql, kotest-assertion-ktor etc along with the junit runner, and assertion module, so that people will know that they need to add those dependencies in their build file.

Good idea. I'll add another page for that.

File: doc/reference.md (hunk 1 above)
Comment (sksamuel) [reply doc/reference.md:120]: The console runner can't be used for gradle, only for the plugin. We can clarify once we release the plugin.

**Pull request merged**